### DISCUSSION TITLE
How can I add this to my docker instance of netbox?

### DISCUSSION THREAD

**Discussion by DuggiFresh** [mattieserver/netbox-topology-views#446] [Q&A]
I tried following the Docker Installation guide, but im having issues getting it to work. I follow it and I put PLUGINS = ["netbox_topology_views"] instead of what's listed in the guide. Everything else that's in the guide I keep the same. I definitely think I'm doing something wrong here. Any help would be much appreciated

**Comment by rossigee** [comment]
In case it helps, this is what works for me:

```
FROM netboxcommunity/netbox:v3.6.9

RUN pip install netbox-topology-views
RUN cp -a /opt/netbox/venv/lib/python3.11/site-packages/netbox_topology_views/static/netbox_topology_views /opt/netbox/netbox/static
```

**Reply by DuggiFresh** [reply]
It installs without errors but doesnt show up for me :/
